From c1bc880ec9708d6de9ae3fc42d77874279ab87b6 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Wed, 11 Jun 2014 12:01:53 +0200
Subject: [PATCH] nm-dispatcher: explicitly add gio/gio.h

Without this header Buildroot's build complains about unknown
types like GFile etc.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 src/nm-dispatcher.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/nm-dispatcher.c b/src/nm-dispatcher.c
index 7a0c8de..336b91f 100644
--- a/src/nm-dispatcher.c
+++ b/src/nm-dispatcher.c
@@ -20,6 +20,7 @@
  */
 
 #include <glib.h>
+#include <gio/gio.h>
 #include <string.h>
 
 #include "nm-dispatcher.h"
-- 
1.7.7

